akeneo-design-system/src/components/Image/Image.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {Ref} from 'react';
+import styled, {css} from 'styled-components';
+
+const ImageContainer = styled.img<{fit: 'cover' | 'contain'; height?: number; width?: number; isStacked: boolean}>`
+  background: white;
```

Comment:
```suggestion
  background: ${getColor('white')};
```
I believe it's a good practice to use this, in case someday the white changes :trollface: